backend/model/userTag.go

```diff
@@ -0,0 +1,24 @@
+package model
+
+import (
+	"gorm.io/gorm"
+)
+
+type UserTag struct {
+	gorm.Model
+	UserId    string
+	TagName   string
+	IsStarred bool
+}
+
+func CreateUserTag(userId string, tagName string, isStarred bool) error {
+	userTag := UserTag{UserId: userId, TagName: tagName, IsStarred: isStarred}
+	result := db.Create(&userTag)
+	return result.Error
+}
+
+func GetUserTag(tagName string) (UserTag, error) {
```

Comment:
```suggestion
func GetUserTagsByUserId(tagName string) (UserTag, error) { ```

↓を参考に、そのユーザーが持ってるタグをすべて取得する、という形にしたいです！
https://gorm.io/ja_JP/docs/query.html